client/components/StatusBar/index.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Alert, Button } from 'react-bootstrap';
+import { Octicon, Octicons } from 'octicons-react';
+import {
+    getConflictsByCycleId,
+    getIssuesByTestId,
+    getResultsByCycleId
+} from '../../actions/cycles';
+
+class StatusBar extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            action: null,
+            icon: 'info',
+            message: '',
+            variant: 'info',
+            visible: false
+        };
+    }
+
+    async componentDidMount() {
+        const { cycle, dispatch, test } = this.props;
+        let { action, icon, message, variant, visible } = this.state;
+
+        await dispatch(getIssuesByTestId(test.id));
+        // TODO: Make these actually do something
+        await dispatch(getConflictsByCycleId(cycle.Id));
+        await dispatch(getResultsByCycleId(cycle.Id));
+
+        if (this.props.issues.length) {
+            action = (
+                <Button
+                    variant="warning"
+                    onClick={this.props.handleRaiseIssueClick}
+                >
+                    Review issues
+                </Button>
+            );
+            message = 'This test has open issues.';
+            variant = 'warning';
+            visible = true;
+        }
+
+        // TODO: Not implemented
+        if (this.props.conflicts.length) {
+            action = (
+                <Button variant="warning" onClick={this.props.handleRedoClick}>
+                    Re-do Test
+                </Button>
+            );
+            message = 'This test has conflicting results.';
+            variant = 'warning';
+            visible = true;
+        }
+
+        // TODO: Not implemented
+        if (this.props.results.length) {
+            action = (
+                <Button variant="info" onClick={this.props.handleCloseRunClick}>
+                    Close
+                </Button>
+            );
+            message = 'This test is complete.';
+            variant = 'info';
+            visible = true;
+        }
```

Comment:
I'm a bit confused by this sequence of steps. So if there is a conflict, but the test is complete, the conflict is not shown in the status bar?

Also, if there is a conflict, I think the option should not be to "redo test" but "raise an issue".

But then who will re-do the test with the correct information once the issue is resolve? Maybe we need to leave the issue in the status bar until the tester marks the issue as "resolved for this test."